File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/01_Super_Easy/12403_:::_Save_Setu/Main.cpp
using namespace std;
#include <cstdio>
#include <cstring>

bool test=true;

int main()
{
	if(test) freopen("in.txt","r",stdin);
	
	char comando[7];
	int casos, total=0, estaDonacion;
	scanf("%d", &casos);
	while(casos--)
	{
		scanf("%s", comando);
		if(strcmp(comando,"donate")==0)
		{
			scanf("%d", &estaDonacion);
			total+=estaDonacion;
		}
		else printf("%d\n", total);
	}

	return 0;
}

File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/01_Super_Easy/12403_:::_Save_Setu/Main.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main
{
	private static BufferedReader br;
	private static boolean test=true;

	public static void main(String[] args) throws IOException
	{
		if(test) br=new BufferedReader(new FileReader(new File("in.txt")));
		else br=new BufferedReader(new InputStreamReader(System.in));
		//-----------------------------------------------------------
		int casos=Integer.parseInt(br.readLine()), total=0;
		String[] comando;

		while(casos--!=0)
		{
			comando=br.readLine().split(" ");
			if(comando[0].equals("donate")) total+=Integer.parseInt(comando[1]);
			else System.out.println(total);
		}		
		//-----------------------------------------------------------
		br.close();
	}
}
